numba/_helperlib.c

```diff
@@ -872,8 +873,13 @@ static void traceback_add(const char *funcname, const char *filename, int lineno
     Py_DECREF(frame);
     return;
 
+#if (PY_MAJOR_VERSION == 3) && (PY_MINOR_VERSION == 12) /* 3.12 */
+error:
+    _PyErr_ChainExceptions1(exc);
+#elif (PY_MAJOR_VERSION == 3) && (PY_MINOR_VERSION <= 11) /* 3.11 and below */
```

Comment:
This does not match the explicit declaration of supported versions pattern used elsewhere ?

Reply:
Fixed in 7d848e1139383e9ed0f68bbc201f4008186c5cd3